pkg/shell/nav.jsx

```diff
@@ -19,10 +19,16 @@ export const SidebarToggle = () => {
     useEffect(() => {
         const handleClickOutside = () => setActive(false);
 
-        document.getElementById("nav-system").classList.toggle("interact");
         window.addEventListener("blur", handleClickOutside);
 
         return () => window.removeEventListener("blur", handleClickOutside);
+    }, []);
+
+    useEffect(() => {
+        if (active)
+            document.getElementById("nav-system").classList.add("interact");
+        else
+            document.getElementById("nav-system").classList.remove("interact");
```

Comment:
This does the same thing, but with 1 line. It either adds or removes, based on the boolean(ish) value of `active` passed as a second option to [classList.toggle()](https://developer.mozilla.org/en-US/docs/Web/API/DOMTokenList/toggle).

```suggestion
        document.getElementById("nav-system").classList.toggle("interact", active);
```

...But it could probably be integrated in the above `useEffect` block as well, right? (There probably wouldn't need to be two `useEffect` blocks?)

Reply:
We need the two useEffect blocks, so that the first one runs only on component's first render, as it adds the event listeners.